src/main/java/teammates/logic/core/FeedbackResponsesLogic.java

```diff
@@ -345,15 +345,15 @@ public boolean isResponseOfFeedbackQuestionVisibleToStudent(FeedbackQuestionAttr
                    && question.giverType.equals(FeedbackParticipantType.STUDENTS); 
                                         
         if (isStudentRecipientType
-            && question.isResponseVisibleTo(FeedbackParticipantType.RECEIVER)) {
+                && question.isResponseVisibleTo(FeedbackParticipantType.RECEIVER)) {
```

Comment:
Should be ok (still under 110 chars for me) to combine this condition into one line